Tools/Create-Azure-Sentinel-Solution/pipeline/createSolutionV4.ps1

```diff
@@ -140,12 +141,28 @@ try
 
 						try {
 							Write-Host "Downloading $finalPath"
-							$rawData = (New-Object System.Net.WebClient).DownloadString($finalPath)
+							$isFilePathPresent = Test-Path -Path "$finalPath"
+							Write-Host "Is $finalPath file path present $isFilePathPresent"
+							if ($isFilePathPresent) {
+								$rawData = (New-Object System.Net.WebClient).DownloadString($finalPath)
+							}
+							else {
+								if ($fileExtension -eq "json" -or $fileExtension -eq "JSON") {
```

Comment:
I would prefer to handle for all extensions if this is the expected case but not only for json files?

Reply:
We have identified this issue for .JSON dataconnectors only as existing files already have caps json extension for data connectors. We will take this as a separate backlog item for now.